File: UWPApp/UWPApp/Model/Record.cs
using Windows.UI.Xaml.Media.Imaging;

namespace UWPApp.Model
{
    public class Record
    {
        public string id;
        public string userId;
        public string title;
        public string content;
        public string image;
        public string audio;
        public string video;
        public string date;
        public string nickname;
        public long favoriteNum;
        public BitmapImage userAvatar;
    }
}

File: Server/model/record.js
module.exports = (mongoose) => {
  const Schema = mongoose.Schema
  const ObjectId = Schema.ObjectId

  const RecordSchema = new Schema({
    date: String,
    title: String,
    image: String,
    audio: String,
    video: String,
    content: String,
    favoriteNum: Number,
    userId: ObjectId
  })
  
  return mongoose.model('record', RecordSchema)
}
